flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlUseCatalog.java

```diff
@@ -35,7 +35,7 @@
  */
 public class SqlUseCatalog extends SqlCall {
 
-	public static final SqlSpecialOperator OPERATOR = new SqlSpecialOperator("USE CATALOG", SqlKind.OTHER);
+	public static final SqlSpecialOperator OPERATOR = new SqlSpecialOperator("USE CATALOG", SqlKind.OTHER_DDL);
```

Comment:
why is this required?

Reply:
Both `validate(sqlNode: SqlNode)` methods in `FlinkPlannerImpl` in flink-planner/blink-planner will validate row type,  SqlKind.OTHER kind will failed the validation.

Reply:
After think twice, I revert the SqlKind and modify the planner validate condition.